packages/bee-q/src/components/dropdown/bq-dropdown.tsx

```diff
@@ -0,0 +1,164 @@
+import { h, Component, Element, Event, EventEmitter, Prop, Listen } from '@stencil/core';
+
+import { FloatingUIPlacement } from '../../services/interfaces';
+
+/**
+ * @part base - The component's internal wrapper.
+ * @part trigger - The `div` element used to display the trigger element
+ * @part panel - The `div` element used to display the panel element (bq-panel)
+ */
+@Component({
+  tag: 'bq-dropdown',
+  styleUrl: './scss/bq-dropdown.scss',
+  shadow: true,
+})
+export class BqDropdown {
+  // Own Properties
+  // ====================
+
+  private trigger: HTMLElement;
+  private panelElement: HTMLBqPanelElement;
+
+  // Reference to host HTML element
+  // ===================================
+
+  @Element() el!: HTMLBqDropdownElement;
+
+  // State() variables
+  // Inlined decorator, alphabetical order
+  // =======================================
+
+  // Public Property API
+  // ========================
+
+  /** Distance (px) between the panel and the trigger element. */
+  @Prop({ reflect: true }) panelDistance?: number = 4;
+
+  /** Position of the panel */
+  @Prop({ reflect: true }) panelPlacement?: FloatingUIPlacement = 'bottom-start';
+
+  /**
+   * If true, panel is visible.
+   * You can toggle this attribute to show/hide the panel.
+   */
+  @Prop({ reflect: true }) panelOpen?: boolean = false;
+
+  /**
+   * Determines whether the scrollbar is visible or hidden within the panel.
+   */
+  @Prop({ reflect: true }) panelScrollbar?: boolean = false;
```

Comment:
I don't think we need a property to set scroll in the panel. Instead, I would recommend having a `panelHeight` property, and if the amount of options is more than the available height, the panel body should scroll automatically.

Reply:
Agree. Will do!

Reply:
Actually this prop does not set scroll in the panel. The panel has a fixed height. If the amount of options is bigger than the height, by default, the panel is scrollable, but the scrollbar is hidden just like for `bq-side-menu`.
`panelScrollbar` will set the scrollbar to visible.
You can check the `With Scrollbar` story from Storybook.

Reply:
Then it is clear we don't need this property 😉